src/fides/api/models/privacy_experience.py

```diff
@@ -175,68 +175,38 @@ class PrivacyExperienceConfigHistory(ExperienceConfigBase, Base):
     )
 
 
-class PrivacyExperienceBase:
-    """Base Privacy Experience fields that are common between privacy experiences and historical records"""
-
-    disabled = Column(Boolean, nullable=False, default=False)
-    component = Column(EnumColumn(ComponentType), nullable=False)
-    region = Column(EnumColumn(PrivacyNoticeRegion), nullable=False, index=True)
-    version = Column(Float, nullable=False, default=1.0)
-
-    # Attribute that can be added as the result of "get_related_privacy_notices". Privacy notices aren't directly
-    # related to experiences.
-    privacy_notices: List[PrivacyNotice] = []
-
-    # Attribute that is cached on the PrivacyExperience object by "get_should_show_banner", calculated at runtime
-    show_banner: bool
-
-
-class PrivacyExperience(PrivacyExperienceBase, Base):
+class PrivacyExperience(Base):
     """Stores Privacy Experiences for a given just a single region.  The Experience describes how to surface
     multiple Privacy Notices to the end user in a given region.
 
     There can only be one component per region.
     """
 
+    component = Column(EnumColumn(ComponentType), nullable=False)
```

Comment:
Component is the last remaining duplicated field between PrivacyExperience and PrivacyExperienceConfig.  There's a lot of logic centered around this field as well as a FK constraint so I propose leaving this one.  We just need to make sure they stay in sync. I do have a prevention from keeping ExperienceConfig component types from changing after they're created which helps a lot.

Reply:
yup, this seems fine. it feels like the `component` is fundamental to both the `PrivacyExperience` and the `ExperienceConfig`. And like you said - it's effectively _fixed_  (immutable) for records of both models.